docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -31,9 +31,9 @@ For OS or distribution specific instructions see below:
 
 apt install build-essential libxkbfile-dev libsecret-1-dev libx11-dev
 
-# Install Node16 (using `nvm` - see above) and enable corepack (for `yarn`)
+# Install Node.js (using `nvm` - see above) and enable corepack (for `yarn`)
 
-nvm install 16
```

Comment:
maybe write `nvm install <version>` instead to make clear that specifying the version is required. The same for the other cases of `nvm install`

Reply:
I don't understand, the entire point of .nvmrc and this PR is so that we no longer need to specify the version in the documentation and instead control it entirely from that one file in the pulsar project:
```
 ~/p/pulsar $  nvm install                                                                    
Found '/home/.../pulsar/.nvmrc' with version <16>
Downloading and installing node v16.18.1...
Downloading https://nodejs.org/dist/v16.18.1/node-v16.18.1-linux-x64.tar.xz...
############################################################################################################################ 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v16.18.1 (npm v8.19.2)
```

Reply:
Oh sorry, I didn't know that nvm uses that version automatically.